server_windows.go

```diff
@@ -33,7 +44,148 @@ func (s *Server) toLocalPath(p string) string {
 			// e.g. "/C:" to "C:\\"
 			return tmp
 		}
+
+		if s.winRoot {
+			// Make it so that "/Windows" is not found, and "/c:/Windows" has to be used
+			return `\\.\` + tmp
+		}
 	}
 
 	return lp
 }
+
+func bitsToDrives(bitmap uint32) []string {
+	var drive rune = 'a'
+	var drives []string
+
+	for bitmap != 0 {
```

Comment:
[invalid value guarding] Something I thought of, we might want also a `drive <= 'z'` … I’m pretty sure windows should never return a set bit higher than the 26th bit, but the guard probably isn’t a bad idea still.